api/src/main/java/org/pmiops/workbench/utils/FieldValues.java

```diff
@@ -0,0 +1,56 @@
+package org.pmiops.workbench.utils;
+
+import com.google.cloud.bigquery.FieldList;
+import com.google.cloud.bigquery.FieldValue;
+import com.google.cloud.bigquery.FieldValue.Attribute;
+import com.google.cloud.bigquery.FieldValueList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.joda.time.DateTime;
+
+/** Utility class for working with FieldValueLists, FieldValues, and Fields */
+public final class FieldValues {
```

Comment:
Nice. BigQueryService is doing something similar to this class. I think we could move this functionality into the BigQueryService and replace it's methods that are very similar, like getString(List<FieldValue> row, int index), getBoolean(List<FieldValue> row, int index).. etc. 
https://github.com/all-of-us/workbench/blob/c516872973fe331af315a0d4ef1f99da76347455/api/src/main/java/org/pmiops/workbench/api/BigQueryService.java#L126-L126

Reply:
Yeah. I kinda like static helpers to live in a class named after the thing, and don't really expect static methods on services. But if you only need this when you already have BigQueryService, that makes sense too.

Reply:
I decided to stick with a utility class so that I can use the methods inside an enum class (which can't inject BigQueryService). There are no dependencies on BigQuery itself.